bin/css-validator

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+`echo eval exec node $(printf %q\  "$0" "$@")`
+const fs = require('fs')
+const validateCss = require('css-validator')
```

Comment:
For the sake of consistency with this repo, please use ES5 syntax (i.e. `var`, full functions).

Also, please use consistent styling (i.e. semicolons, braces on `if` statements)

It might help to add `bin/` to our `lint` script in `package.json`

Also, for loading `validateCss`, we'll want to load `require('../')`